common/src/main/java/com/thekillerbunny/goofyplugin/mixin/FiguraGlobalsDocsMixin.java

```diff
@@ -0,0 +1,12 @@
+package com.thekillerbunny.goofyplugin.mixin;
+
+import com.thekillerbunny.goofyplugin.lua.GoofyAPI;
+import org.figuramc.figura.lua.docs.FiguraGlobalsDocs;
+import org.figuramc.figura.lua.docs.LuaFieldDoc;
+import org.spongepowered.asm.mixin.Mixin;
+
+@Mixin(value = FiguraGlobalsDocs.class, remap = false)
+public class FiguraGlobalsDocsMixin {
+    @LuaFieldDoc("globals.goofy")
+    public GoofyAPI goofy;
+}
```

Comment:
no ending EOL; some programs will complain
```suggestion
}
```